### PR TITLE
Dynamic vcf parse

### PR DESCRIPTION
**Pull request by adrosenbaum**
Problem:
the information given in the INFO column in a vcf might be different from file to file. Currently, mutacc assumes what IDs might be in the INFO column, and parses the values based on a assumed format. This is an issue, if it is decided that the information from an additional ID field is to be included in the database, this must be changed in the code. Some formats might require some instructions on how they should be parsed, .e.g. the value might be a array, where each element is separated by a '|' symbol. There should be a way for the user to specify dynamically in a file what information from the INFO column that should be included in the database, and instructions on how these values should be parsed.

Solution:
This PR makes it possible what IDs in the INFO column of a vcf that should be included in the database. The user now have the option to specify what IDs, and how, these fields are parsed from the vcf, by specifying this information in a yaml file. In this way, information from the vcf can be included dynamically, without assuming specific vcf formats in the code.

**How to test**:
1. install on stage: `bash servers/resources/hasta.scilifelab.se/update-mutacc-stage.sh dynamic-vcf-parse`
1. activate stage: `us`
1. run following command: `cg upload process-solved --case-id <case_id>`

**Expected outcome**:
The uploaded variants should contain the same information as before. 

**Review:**
- [x] code approved by @moonso 
- [x] tests executed by @adrosenbaum 
- [x] "Merge and deploy" approved by @ingkebil 
Thanks for filling in who performed the code review and the test!

This is minor **version bump** because a feature is added in a backwards compatible manner.

### CODE REVIEW

**Comment by northwestwitch** [comment]
I'll not have much time to look at this this week, but I can do it on the next

**Comment by adrosenbaum** [comment]
> I'll not have much time to look at this this week, but I can do it on the next

Ok thank you! I really need someone to review my PRs in this repo in the future 😄

File: mutacc/builds/build_case.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class Case(dict):
     """
         Class with methods for handling case objects
     """
-    def __init__(self, input_case, read_dir, padding=None, picard_exe=None):
+    def __init__(self, input_case, read_dir, padding=None, picard_exe=None, vcf_parse=None):
```
Comment (moonso) [inline mutacc/builds/build_case.py]: Could you describe these arguments in the docstring?

File: mutacc/builds/build_case.py [hunk 2]
```diff
@@ -45,7 +46,7 @@ def __init__(self, input_case, read_dir, padding=None, picard_exe=None):
         # Build case
         self['case'] = self.input_case['case']
 
-    def _build_variants(self, padding=None, rank_model_version=None):
+    def _build_variants(self, padding=None, rank_model_version=None, vcf_parse=None):
```
Comment (moonso) [inline mutacc/builds/build_case.py]: Same here, be more explicit in the docstring. Is rank_model_version str or float etc

**Comment by adrosenbaum** [comment]
> Great @adrosenbaum !

Thanks for the review @moonso

**Comment by adrosenbaum** [comment]
@ingkebil can you approve for merge and deploy? 😄

**Comment by ingkebil** [comment]
done!

**Pull request merged**